### PR TITLE
Sprint2 professors page

### PR DESCRIPTION
**Pull request by kranthijella**
Merging the Professor Page into the Main below are the task  completed .
1. Updated the test cases.
2. Added Professor Screen.
3. Code Coverage.
4. Bug Fixes.

### CODE REVIEW

**Comment by HanSolo-P** [comment]
lgtm

File: class_critique_app/lib/operations/auth_service.dart [hunk 1]
```diff
@@ -1,61 +1,195 @@
...
+  }
+
+  Future<UserCredential?> loginUser(String email, String password) async {
     return await _login(email, password);
   }
 
-  Future<String> _createAccount(String email, String password) async {
+  Future<String> saveUser(
+      String email, String fullName, String universityName) async {
+    //print('******* Step 6: saveUser called');
```
Comment (swatkat20) [inline class_critique_app/lib/operations/auth_service.dart]: Kindly remove unwanted debug prints

File: class_critique_app/lib/screens/login_screen.dart [hunk 2]
```diff
@@ -1,13 +1,17 @@
 // ignore_for_file: prefer_const_constructors
```
Comment (HanSolo-P) [inline class_critique_app/lib/screens/login_screen.dart:1]: can u remove this comment if it is unnecessary

File: class_critique_app/lib/screens/login_screen.dart [hunk 3]
```diff
@@ -93,6 +99,7 @@ class _LoginScreenState extends State<LoginScreen> {
                     SizedBox(height: 16),
                     // Password Text Field
```
Comment (HanSolo-P) [inline class_critique_app/lib/screens/login_screen.dart:100]: pls remove this comment too

**Review by abhi-s7 — APPROVED** [review]

**Review by HanSolo-P — COMMENTED** [review]
lgtm

**Pull request merged**